deploy/olm-catalog/integreatly-operator/2.6.1/integreatly-operator.v2.6.1.clusterserviceversion.yaml

```diff
@@ -47,6 +47,7 @@ metadata:
     support: RHMI
   name: integreatly-operator.v2.6.1
   namespace: placeholder
+  olm.skipRange: '>=2.4.0 <2.6.1'
```

Comment:
Is this going to replace anything greater than or equal to 2.4.0? Any reason we are doing that?

Reply:
Just noting here that we usually clear up old manifests for products between releases to get round the configmap limitations, only leaving the minimum required to upgrade from the previous version. This assumes that we always apply every release of our operator. Assuming nothing was upgraded between these versions, we might be ok, but if this is something we plan on doing more often, or want to allow for, then we need to maintain the correct OLM graph for every product.

Reply:
@mikenairn Yeah this is a unique case. Unfortunately due to issues with 2.5.0 upgrades in production, SRE put a halt to them and so we have clusters on both 2.4.0 and 2.5.0. We want to be able to upgrade both directly to 2.6.1 which is why we need to specify this range

Reply:
@mikenairn We'll be looking to remove this following the 2.6.1 upgrade. We just need it here for the reasons I've listed above